tests/IntegrationTests/WebAppCallsApiCallsGraphUiTests/TestingWebAppCallsApiCallsGraphLocally.cs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Identity.Lab.Api;
+using Microsoft.Playwright;
+using WebAppUiTests;
+using Xunit;
+using Xunit.Abstractions;
+using Process = System.Diagnostics.Process;
+
+namespace WebAppCallsApiCallsGraphUiTests
+{
+#if !FROM_GITHUB_ACTION
+    [CollectionDefinition(nameof(TestingWebAppCallsApiCallsGraphLocally), DisableParallelization = true)] // since this changes environment variables we don't want it running at the same time as other tests
+    public class TestingWebAppCallsApiCallsGraphLocally
+    {
+        private const string LocalhostUrl = @"https://localhost:";
+        private const string DevAppPath = @"DevApps\WebAppCallsWebApiCallsGraph";
+        private const string TodoListServicePath = @"\TodoListService";
+        private const string TodoListClientPath = @"\Client";
+        private const string GrpcPath = @"\gRPC";
+        private const string TodoListServiceExecutable = @"\TodoListService.exe";
+        private const string TodoListClientExecutable = @"\TodoListClient.exe";
+        private const string GrpcExecutable = @"\grpc.exe";
+        private const uint TodoListServicePort = 44350;
+        private const uint TodoListClientPort = 44321;
+        private const uint GrpcPort = 5001;
+        private const string SignOutPagePath = @"/MicrosoftIdentity/Account/SignedOut";
+        private const string TodoTitle1 = "Testing create todo item";
+        private const string TodoTitle2 = "Testing edit todo item";
+        //private const string ManagedIdentityObjectId = "9c5896db-a74a-4b1a-a259-74c5080a3a6a";
+        private string UiTestAssemblyLocation = typeof(TestingWebAppCallsApiCallsGraphLocally).Assembly.Location;

```

Comment:
mark as readonly and should be `_uiTestAssem.....` as it's a field

Reply:
is this in the editor config?

Reply:
what editor config do you mean @keegan-caruso